mlir/utils/performance/createPerformanceReports.py

```diff
@@ -13,11 +13,26 @@ def printAllPerformance(chip, lib='rocBLAS'):
 
     try:
         df = pd.read_csv(chip + '_' + reportUtils.PERF_REPORT_FILE[lib])
+        quickDf = pd.read_csv('quick_'  + chip + '_' + reportUtils.PERF_REPORT_FILE[lib])
```

Comment:
Note remains:
1. This script must not fail if there is no quick tuning perf file.
2. These columns should be in the main dataframe anyway.
3. If you decide you have to have a seperate `quick_` file, that filename needs to be configurable.

Reply:
I changed the approach to expand the perfRunner with **--quick-tuning-db** . I'll update the PR soon.